lib/datadog/kit/identity.rb

```diff
@@ -56,6 +56,11 @@ def self.set_user(trace, id:, email: nil, name: nil, session_id: nil, role: nil,
         others.each do |k, v|
           trace.set_tag("usr.#{k}", v) unless v.nil?
         end
+
+        if ::Datadog::AppSec.enabled?
+          user = OpenStruct.new(id: id)
```

Comment:
@lloeki I'm creating a dummy object for now. I would work on having the set of objects we discussed on this other [PR](https://github.com/DataDog/dd-trace-rb/pull/2627#discussion_r1111807670) on a separate PR.

For now `OpenStruct` allows to move fast 💨

Reply:
Absolutely. I'm not a fun of `OpenStruct` but for getting out of the way is very handy